internal/commands/push/command.go

```diff
@@ -54,14 +65,36 @@ func (cmd *Command) Flags() []flags.Flag {
 				},
 			},
 		},
+		flags.BoolFlag{
+			Value: &cmd.inputs.IncludeNodeModules,
+			Meta: flags.Meta{
+				Name: flagIncludeNodeModules,
+				Usage: flags.Usage{
+					Description: "Export and include Realm app dependencies from a node_modules archive",
```

Comment:
copy/paste usage string here
```suggestion
					Description: "Import and include Realm app dependencies from a node_modules archive",
```